src/ObjectUtils/LinuxMap.cpp

```diff
@@ -18,15 +18,70 @@
 #include <type_traits>
 #include <utility>
 
+#include "ObjectUtils/CoffFile.h"
 #include "ObjectUtils/ElfFile.h"
+#include "ObjectUtils/ObjectFile.h"
 #include "OrbitBase/Logging.h"
 #include "OrbitBase/ReadFileToString.h"
 
 namespace orbit_object_utils {
 
 using orbit_grpc_protos::ModuleInfo;
+using orbit_object_utils::CoffFile;
+using orbit_object_utils::CreateCoffFile;
 using orbit_object_utils::CreateElfFile;
 using orbit_object_utils::ElfFile;
+using orbit_object_utils::ObjectFile;
+
+ErrorMessageOr<ModuleInfo> CreateModule(
```

Comment:
This overload should be either be made `static` or also declared in `LinuxMap.h`

Reply:
New version doesn't have the overload anymore.